pkg/container/vector/vector.go

```diff
@@ -58,6 +58,9 @@ type Vector struct {
 	// TODO: check whether isBin should be changed into array/bitmap
 	// now we assumpt that it can only be true in the case of only one data in vector
 	isBin bool
+
+	// idx for low cardinality scenario.
+	idx any
```

Comment:
any?

Reply:
If you use `idx *index.LowCardinalityIndex` directly, it will cause import cycle problem because LowCardinalityIndex need package `vector`. So solve this problem by `any`, and assert it as `LowCardinalityIndex` when you use it.